### PR TITLE
add function unlike to cancel like

### PR DESCRIPTION
**Pull request by oiz5201618**

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -370,6 +373,36 @@ static Response *post(Request *req)
...
+        char sbuff[1024];
+        sprintf(sbuff, "/profile/%d/", post->authorId);
+        bsDel(idStr);
+        return responseNewRedirect(sbuff);
+    }
+
+fail:
+    bsDel(idStr);
+    return responseNewRedirect("/dashboard/");
+
```
Comment (jserv) [inline src/main.c]: Don't leave the blank line.

File: src/models/like.c [hunk 2]
```diff
@@ -44,6 +44,33 @@ Like *likeCreate(sqlite3 *DB, int accountId, int authorId, int postId)
...
+    rc = sqlite3_prepare_v2(DB,
+			    "DELETE"
+			    " FROM likes"
+			    " WHERE account = ?"
+			    "   AND post = ?",
+			    -1, &statement, NULL);
+
+    if (rc != SQLITE_OK) return NULL;
+    if (sqlite3_bind_int(statement, 1, accountId) != SQLITE_OK) goto fail;
+    if (sqlite3_bind_int(statement, 2, postId) != SQLITE_OK) goto fail;
```
Comment (jserv) [inline src/models/like.c]: You can rewrite the two lines as following:

```
if ((sqlite3_bind_int(statement, 1, accountId) != SQLITE_OK) ||
    (sqlite3_bind_int(statement, 2, postId) != SQLITE_OK))
    goto fail;
```

It is more readable.

Comment (oiz5201618) [inline src/models/like.c]: Teacher, the likeCreate function( in like.c line19 ) also write like this

`if (sqlite3_bind_int(statement, 1, t) != SQLITE_OK) goto fail;`
`if (sqlite3_bind_int(statement, 2, accountId) != SQLITE_OK) goto fail;`
`if (sqlite3_bind_int(statement, 3, authorId) != SQLITE_OK) goto fail;`
`if (sqlite3_bind_int(statement, 4, postId) != SQLITE_OK) goto fail;`

Could I rewrite it ?

**Comment by jserv** [comment]
Use `git rebase -i` to squash the two commits.

**Comment by oiz5201618** [comment]
Sorry, I pushed the wrong commit.
I have fixed it.

**Pull request merged**